presto-hive/src/test/java/io/prestosql/plugin/hive/metastore/cache/TestCachingHiveMetastore.java

```diff
@@ -285,6 +286,18 @@ public void testGetPartitionStatistics()
         assertEquals(mockClient.getAccessCount(), 3);
     }
 
+    @Test
+    public void testUpdatePartitionStatistics()
```

Comment:
add a commit that adds this test as a first commit.
Then you will be able to show improvement with `Remove getTable ...` commit

Reply:
I ran the test without the removal of getTable call, it did 5 calls to the metastore. The new one does 4 calls. The test failed with the following error : 

`java.lang.AssertionError: expected [5] but found [6]
Expected :5
Actual   :6`

Reply:
Please make `Test CachingHiveMetastore update partition statistics ` test commit to be before than `Remove getTable call from updatePartitionStatistics` commit

Reply:
The test commit is before the removal commit in my git log. I think github shows commit in chronology of time + I did a rebase of test commit due to some conflicts. So, github is showing the test commit later. will rewrite the removal commit as well for consistency.

Reply:
So could you write the test in such way that it shows an improvement in `removal` commit? Maybe do two `update` calls?

Reply:
I have added the test commit first along with the expected number of calls in that to be 5 (without the removal change). In the removal commit, I've updated the test to expect 4 calls (changing the test) - so that it shows improvement over the earlier test. Does that suffice?

Reply:
I would expect that https://github.com/prestosql/presto/pull/2734/commits/1c45b21dcacbcb9b37d9aa4643f214c4dda42b6b shows reduced access count in `testUpdatePartitionStatistics`

Reply:
[test reduced count](https://github.com/prestosql/presto/commit/1c45b21dcacbcb9b37d9aa4643f214c4dda42b6b#diff-362b72e724f9b4c14efe898b705f547dR302) - shows the reduced access count in `testUpdatePartitionStatistics` in the [1c45b21](https://github.com/prestosql/presto/commit/1c45b21dcacbcb9b37d9aa4643f214c4dda42b6b) commit.